server/pulp/server/content/web/views.py

```diff
@@ -128,6 +127,13 @@ def redirect(request, key):
     def __init__(self, **kwargs):
         super(ContentView, self).__init__(**kwargs)
         self.key = Key.load(pulp_conf.get('authentication', 'rsa_key'))
+        # Make sure all requested paths fall under these sub-directories, otherwise
+        # we might find ourselves serving private keys to all and sundry.
+        self.safe_serving_paths = [os.path.realpath(os.path.join(LOCAL_STORAGE, subdir))
```

Comment:
Is this setting excluded from our configuration[0] on purpose? I can't imagine why it would be excluded, but it's not in our default configuration file. It is in the default settings dictionary, though.

[0] https://github.com/pulp/pulp/blob/master/server/etc/pulp/server.conf#L78